app/controllers/content_items_controller.rb

```diff
@@ -24,6 +24,35 @@ class ContentItemsController < ApplicationController
   def show
     load_content_item
 
+    # TEMPORARY (author: richard.towers, expected end date: November 30 2023)
+    # Content specific AB test for the Find your UTR number page
+    if @content_item.base_path == "/find-utr-number"
+      ab_test = GovukAbTesting::AbTest.new(
+        "find_utr_number_video_links",
+        dimension: 300, # TODO: which dimension should we use?
+        allowed_variants: %w[HelpText VideoLink],
```

Comment:
I might be getting confused here - but is there not something about having A,B and Z variants? And the control is Z. I'll see if I can dig out some docs.

Reply:
I checked with @tarastockford who linked me to [this](https://docs.google.com/document/d/1MAFk0JYh9wWs-bfkmhHv8V9RPpZL3kU7uS_6GLJoed8/edit#heading=h.ig4gd5kwqtxf) and here's some [prior art](https://github.com/alphagov/collections/pull/2711/files#diff-a80f7fd7d691b7ddbcccd6c4da7abf14d1c93fa68d0a42986f4397df30957427R4)